### DISCUSSION TITLE
Output tab is separated from Result, any way to move back?

### DISCUSSION THREAD

**Discussion by pavel-tarasov** [dbeaver/dbeaver#12191] [Q&A]
![Screenshot from 2021-04-21 10-57-26](https://user-images.githubusercontent.com/2952566/115518029-6ea17f00-a290-11eb-9756-08afc9089796.png)
I thought that this question was discussed in https://github.com/dbeaver/dbeaver/discussions/11983
I unchecked this box, but "Ouput" and "Execution Log" tabs still appear on the right of the script and not near to Result tab. Is there a way to keep all these three tabs together under script?

**Comment by sknextcode** [comment]
Seems to be part of the editor window as for now. Don't see a way to add them to the result area or drag and drop them to another slot.

**Reply by tmbrye** [reply]
I just came looking for the same thing.  The execution log use to be in the results area and it is much easier to look at when it is wider instead of now where it pops up on the right of the editor windows.
